bootstrap.sh

```
@@ -1 +1,7 @@
 #!/bin/bash
+kubectl apply -f .infrastructure/namespace.yml
+kubectl apply -f .infrastructure/pv.yml
+kubectl apply -f .infrastructure/pvc.yml
+kubectl apply -f .infrastructure/deployment.yml
+kubectl apply -f .infrastructure/service.yml
+echo "All resources completed."
```

Consider making the script fail fast by adding `set -euo pipefail` near the top and using `kubectl wait` for resources to become ready. Right now the script will continue after errors, which can mask failures. Also ensure the script file has the executable bit set in the repo so reviewers can run it directly. (operational suggestion)